x-pack/plugins/security_solution/server/lib/detection_engine/rule_preview/api/preview_rules/route.ts

```diff
@@ -237,6 +239,8 @@ export const previewRulesRoute = (
               createdAt: new Date(),
               createdBy: username ?? 'preview-created-by',
               producer: 'preview-producer',
+              consumer: SERVER_APP_ID,
```

Comment:
Does this consumer affect behaviour, or it's just type thing here?

Reply:
This field was previously added with the `convertCreateAPIToInternalSchema` function. Now that I have split that function into multiple single-purpose functions, we have to add it manually here. There should be no changes in the resulting rule passed to Alerting.